<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOptions</class>
 <widget class="QDialog" name="DialogOptions">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>387</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>536</width>
    <height>236</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DefaultReset_Button">
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="2">
       <widget class="QPushButton" name="BackupPath_Button">
        <property name="text">
         <string>Choose</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Option4_BackupPath"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The location where we will save our backed up files at. By default, it will read the &lt;span style=&quot; font-weight:600;&quot;&gt;/translator_auto_backup&lt;/span&gt; folder under Zombie Panic! Source Mod Tools root directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Backup Path:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Main Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Option1_MainPath"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The main path for opening files. By default, it will read the &lt;span style=&quot; font-weight:600;&quot;&gt;zps/translations&lt;/span&gt; folder under Zombie Panic! Source root install directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="MainPath_Button">
        <property name="text">
         <string>Choose</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QLineEdit" name="Option2_ExportPath"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The location where we will export our Json Language files. By default, it will read the &lt;span style=&quot; font-weight:600;&quot;&gt;zps/translations&lt;/span&gt; folder under Zombie Panic! Source root directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Export Path:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="ExportPath_Button">
        <property name="text">
         <string>Choose</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QFrame" name="SaveRate">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QSlider" name="Option3_SaveTimer">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="sliderPosition">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Backup Timer</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="Option4_Backup_Timer">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Auto Save Timer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="Option_MinimizeSave">
        <property name="text">
         <string>Minimize the saved .json and .jlang files (Take less time to load in-game)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Auto Save Timer&lt;/span&gt; and &lt;span style=&quot; font-style:italic;&quot;&gt;Backup Timer&lt;/span&gt; counts in &lt;span style=&quot; font-weight:600;&quot;&gt;minutes&lt;/span&gt;, so each tick represent 1 minute. It goes from &lt;span style=&quot; font-weight:600;&quot;&gt;1&lt;/span&gt; up to &lt;span style=&quot; font-weight:600;&quot;&gt;10&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="Option4_BackupOption">
        <property name="text">
         <string>Save the backup file as a .jlang instead of .json</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DefaultReset_Button</sender>
   <signal>pressed()</signal>
   <receiver>DialogOptions</receiver>
   <slot>ResetToDefult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainPath_Button</sender>
   <signal>pressed()</signal>
   <receiver>DialogOptions</receiver>
   <slot>ChooseMain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExportPath_Button</sender>
   <signal>pressed()</signal>
   <receiver>DialogOptions</receiver>
   <slot>ChooseExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BackupPath_Button</sender>
   <signal>released()</signal>
   <receiver>DialogOptions</receiver>
   <slot>ChooseBackup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ResetToDefult()</slot>
  <slot>ChooseMain()</slot>
  <slot>ChooseExport()</slot>
  <slot>ChooseBackup()</slot>
  <slot>UpdateBackupOption(bool)</slot>
  <slot>UpdateMinimize(bool)</slot>
 </slots>
</ui>
